README.md

```diff
@@ -18,8 +18,84 @@ npm install geckoboard
 
 ## Use
 
-The latest documentation and user guide can be found on the Geckoboard developer docs (https://developer.geckoboard.com/)
+The latest documentation and user guide can be found on the Geckoboard developer docs (https://developer.geckoboard.com/) 
 
+
+### API requests
+
+#### Find or create a new dataset
+
+```
+import Geckoboard from 'geckoboard';
+
+const API_KEY = 'YOUR_API_KEY';
```

Comment:
🏄 maybe the example should show getting the API key from an environment variable?